cloud_governance/policy/aws/empty_roles.py

```diff
@@ -1,54 +1,54 @@
 from cloud_governance.common.logger.init_logger import logger
-from cloud_governance.policy.policy_operations.aws.zombie_non_cluster.run_zombie_non_cluster_policies import NonClusterZombiePolicy
+from cloud_governance.policy.helpers.aws.aws_policy_operations import AWSPolicyOperations
 
 
-class EmptyRoles(NonClusterZombiePolicy):
+class EmptyRoles(AWSPolicyOperations):
     """
     This class sends an alert mail for empty role to the user after 4 days and delete after 7 days.
     """
 
+    RESOURCE_ACTION = 'Delete'
+    IAM_GLOBAL_REGION = 'us-east-1'
+
     def __init__(self):
         super().__init__()
 
-    def run(self):
-        """
-        This method returns all empty roles, delete if dry_run no
-        @return:
+    def run_policy_operations(self):
         """
-        return self.__delete_empty_roles()
```

Comment:
return self.__delete_empty_roles() - is it ok that we remove it ?

Reply:
yes, this is the old method

Reply:
@athiruma , Pls fix conflicts

Reply:
Done